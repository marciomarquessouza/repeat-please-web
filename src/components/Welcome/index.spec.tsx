import React from 'react';
import { shallow } from 'enzyme';
import { Welcome } from '.';

const setup = () => shallow(<Welcome />);

describe('#Welcome', () => {
	it('renders properly', () => {
		const wrapper = setup();
		expect(wrapper).toMatchSnapshot();
	});
});
