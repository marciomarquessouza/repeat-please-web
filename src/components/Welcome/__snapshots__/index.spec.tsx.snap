// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Welcome renders properly 1`] = `
<Fragment>
  <styled.div>
    <LogoWelcome />
  </styled.div>
  <styled.div>
    <Component
      data-test="start-now"
      onClick={[Function]}
      variant="secondary"
    >
      startNow
    </Component>
  </styled.div>
</Fragment>
`;
